cidc_schemas/schemas/templates/metadata/ihc_template.json

```
@@ -7,143 +7,115 @@
-                  "merge_pointer": "0/percentage_tumor_positive",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/percentage_tumor_positive"
-                },
-                "tumor positive intensity": {
-                  "merge_pointer": "0/tumor_positive_intensity",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/tumor_positive_intensity"
-                },
-                "average tumor marker intensity": {
-                  "merge_pointer": "0/average_tumor_marker_intensity",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/average_tumor_marker_intensity"
-                },
-                "percent inflammation marker positive": {
-                  "merge_pointer": "0/percent_inflammation_marker_positive",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/percent_inflammation_marker_positive"
-                },
-                "average inflammation marker intensity": {
-                  "merge_pointer": "0/average_inflammation_marker_intensity",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/average_inflammation_marker_intensity"
-                },
-                "clinically positive": {
-                  "merge_pointer": "0/clinically_positive",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/clinically_positive"
-                }
-              },
-
-              "H and E": {
-                "percentage viable tissue": {
-                  "merge_pointer": "0/percentage_viable_tissue",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/percentage_viable_tissue"
-                },
-                "percentage tumor": {
-                  "merge_pointer": "0/percentage_tumor",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/percentage_tumor"
-                },
-                "degree of lymphoid infiltrate": {
-                  "merge_pointer": "0/degree_lymphoid_infiltrate",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/degree_lymphoid_infiltrate"
-                },
-                "percentage fibrosis": {
-                  "merge_pointer": "0/percentage_fibrosis",
-                  "type_ref": "assays/components/imaging/ihc_entry.json#properties/percentage_fibrosis"
+                "data_columns": {
+                    "Sample": {
+                        "cimac ID": {
+                            "merge_pointer": "0/cimac_id",
+                            "type_ref": "sample.json#properties/cimac_id"
+                        },
+                        "IHC image": {
+                            "merge_pointer": "0/files/ihc_image",
+                            "is_artifact": 1,
+                            "gcs_uri_format": "{protocol identifier}/{cimac id}/ihc/ihc.tiff",
```

As I'm moving now the assay prefix in-front of cimac_id in all other assay templates, to be able to issue permissions with prefix like "trial/assay" 
(see here https://github.com/CIMAC-CIDC/cidc-schemas/pull/257) 
I would suggest 
```suggestion
                            "gcs_uri_format": "{protocol identifier}/ihc/{cimac id}/ihc.tiff",
```